spec/features/item_spec.rb

```diff
@@ -28,6 +28,19 @@
     expect(page.find(".alert")).to have_content "updated"
   end
 
+  scenario "User sets a distribution quantity and package size" do
+    item = create(:item)
+    visit url_prefix + "/items/#{item.id}/edit"
+    fill_in "item_distribution_quantity", with: "75"
+    fill_in "item_package_size", with: "50"
+    click_button "Save"
+
+    find("[data-item-id=#{item.id}] [href$=edit]").click
+
+    expect(page).to have_selector("input#item_distribution_quantity[value='75']")
+    expect(page).to have_selector("input#item_package_size[value='50']")
+  end
+
```

Comment:
I'm not sure why this file is still around, but can you please move this example over to `/spec/system/item_system_spec.rb`?

Reply:
@seanmarcia is going to take care of this so I'll approve the branch!